tests/transformations/raise_and_duplicate_test.py

```diff
@@ -0,0 +1,62 @@
+import dace
+from dace.sdfg import nodes
+from dace.transformation.interstate import IfRaising, StateReplication
+from dace.transformation.dataflow import OTFMapFusion
+import numpy as np
+
+
+def test_raise_and_duplicate_and_fusions():
+    N = dace.symbol('N', dace.int64)
+    @dace.program
+    def program(flag: dace.bool, in_arr: dace.float64[N], arr: dace.float64[N]):
+        tmp1 = np.empty_like(arr)
+        tmp2 = np.empty_like(arr)
+        for i in dace.map[0:N]:
+            tmp1[i] = in_arr[i]
+        if flag:
+            for i in dace.map[0:N]:
+                tmp2[i] = tmp1[i]
+        else:
+            for i in dace.map[0:N]:
+                tmp2[i] = tmp1[i]
+        for i in dace.map[0:N]:
+                arr[i] = tmp2[i]
+
+    sdfg = program.to_sdfg()
+    sdfg.apply_transformations([IfRaising, StateReplication])
+    sdfg.simplify()
+    sdfg.apply_transformations_repeated([OTFMapFusion])
+
+    assert len(sdfg.nodes()) == 4
+    assert sdfg.start_state.is_empty()
```

Comment:
the tests fails here but it works on my machine. Is there something specific to the setup in the pipeline?